pkg/minikube/registry/drvs/qemu2/qemu2.go

```diff
@@ -117,7 +117,8 @@ func qemuVersion() (semver.Version, error) {
 	if err != nil {
 		return semver.Version{}, err
 	}
-	v := strings.Split(strings.TrimPrefix(string(rr), "QEMU emulator version "), "\n")[0]
+	v := strings.Fields(strings.Split(strings.TrimPrefix(string(rr), "QEMU emulator version "), "\n")[0])[0]
```

Comment:
```suggestion
	v := strings.Fields(strings.TrimPrefix(string(rr), "QEMU emulator version "))[0]
```
We can remove strings.Split as it's redundant now that you're grabbing the first value of strings.Fields.